core/src/pages.rs

```diff
@@ -186,7 +186,7 @@ impl<const SIZE: u32> PartialOrd<PagesAmount<SIZE>> for Page<SIZE> {
     Default,
     derive_more::Into,
 )]
-pub struct Page<const SIZE: u32>(u32);
+pub struct Page<const SIZE: u32>(pub(crate) u32);
```

Comment:
The same  unsafe

Reply:
Fixed, added method `[cfg(test)] fn into_inner() -> u32`

Reply:
I cannot understand what the reason to add this method ? you can use `u32::from(page)`

Reply:
Missed it, fixed